src/scripts/main.js

```diff
@@ -1 +1,20 @@
 'use strict';
+
+const galleryList = document.querySelector('.gallery__list');
+const largeImg = document.querySelector('.gallery__large-img');
+
+galleryList.addEventListener('click', (e) => {
+  const listItem = e.target.closest('a');
+
+  if (!listItem) {
+    return;
+  }
+
+  showThumbnail(listItem.href, listItem.title);
+  e.preventDefault();
+});
+
+function showThumbnail(href, title) {
```

Comment:
this function would make sense if be reused a few times, now you can just write func body right on 13 line